BayesianOptimization/discussion_parameters.md

```diff
@@ -0,0 +1,46 @@
+## Options for Bayesian Optimization
```

Comment:
These are specific of botorch right? If yes, mention it and add a link to the botorch documentation.